docs/user-guide/container-environment.md

```diff
@@ -81,4 +83,23 @@ Hook handlers are the way that hooks are surfaced to containers.  Containers ca
 
    * HTTP - Executes an HTTP request against a specific endpoint on the container.
 
-[1]: http://man7.org/linux/man-pages/man2/gethostname.2.html
\ No newline at end of file
+[1]: http://man7.org/linux/man-pages/man2/gethostname.2.html
+
+### Debugging Hook Handlers
+
+Currently, the logs for a hook handler are not exposed by default. However, if your handler fails for some reason, it will emit an event. For `PostStart`, this is the `FailedPostStartHook` event. For `PreStop` this is the `FailedPreStopHook` event. An example of this in action is below:
```

Comment:
The logs are posted as an event so whatever the Hook writes to stdout will be visible in the events section of kubectl describe pods <pod>